adapter/pkg/mgw/mgw.go

```diff
@@ -73,7 +81,24 @@ const grpcMaxConcurrentStreams = 1000000
 func runManagementServer(server xdsv3.Server, enforcerServer xdsv3.Server, port uint) {
 	var grpcOptions []grpc.ServerOption
 	grpcOptions = append(grpcOptions, grpc.MaxConcurrentStreams(grpcMaxConcurrentStreams))
-	grpcServer := grpc.NewServer()
+
+	cert, err := tlsutils.GetServerCertificate()
```

Comment:
Exit the code if error pops

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/1582/commits/b804e40a426ab4a0c8f06cc8c5d68598fbfc8882